include/mbedtls/check_config.h

```diff
@@ -281,15 +281,6 @@
 #error "MBEDTLS_PK_PARSE_C defined, but not all prerequisites"
 #endif
 
-#if defined(MBEDTLS_PKCS12_C) && !defined(MBEDTLS_CIPHER_C)
-#error "MBEDTLS_PKCS12_C defined, but not all prerequisites"
```

Comment:
Commenting here because github doesn't let me comment in the right place: we want to adjust the `full_no_cipher*` components as well.

Pre-existing and hardly related, but here it is: looking again at those components, I don't understand why we disabled `MBEDTLS_CTR_DRBG_C` or even `MBEDTLS_DES_C` in `common_test_full_no_cipher_with_psa_crypto`.

Reply:
> Pre-existing and hardly related, but here it is: looking again at those components, I don't understand why we disabled `MBEDTLS_CTR_DRBG_C` or even `MBEDTLS_DES_C` in `common_test_full_no_cipher_with_psa_crypto`.

I don't remember either, so I think I can check if I can add them back.